input/docs/integrations/editors/rider/index.cshtml

```diff
@@ -0,0 +1,28 @@
+Order: 40
+Title: Rider
+Description: Extensions and supported features for JetBrains Rider
+RedirectFrom: docs/editors/rider
+---
+
+<div>
+    DOT NOT MERGE <br />
+    DOT NOT MERGE <br />
+    DOT NOT MERGE -- The following link (.../marketplace) needs to be replaced with the correct link to the plugin in the marketplace<br />
+    DOT NOT MERGE <br />
+    DOT NOT MERGE <br />
+
+    The <a href="https://plugins.jetbrains.com/marketplace" target="_blank">Cake plugin for Rider </a>
+    brings the following features to JetBrains Rider:
+    <ul>
+        <li><a href="/docs/integrations/editors/rider/tasks-window">Cake script and tasks recognition</a> in a tool window</li>
+        <li><a href="/docs/integrations/editors/rider/run-configurations">Run configurations</a></li>
+    </ul>
+</div>
+
+<h1>Installation & configuration</h1>
+
+See <a href="https://plugins.jetbrains.com/marketplace" target="_blank">Cake plugin for Rider</a> for instructions how to install and configure the plugin.
```

Comment:
Does it require a min version of Rider? If so add a annotation like done [here](http://localhost:5080/docs/integrations/editors/visualstudio/#installation-configuration)